components/EventPartners.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Photo from './Photo';
+
+const EventPartners = ({ event }) => {
+  return (<section className="mb-6">
+    <div className="flex flex-row flex-wrap justify-center items-center">
+      {event.partners && event.partners.map(partner => partner.photoUrl && <a href={partner.url || '#'} target="_blank" rel="noreferrer" key={partner.name} className="mr-3">
+        <Photo id={partner.photo} photoUrl={partner.photoUrl} className="w-32 h-16" title={partner.name} />
+      </a>)}
+    </div>
+    {
+    /* { (isAuthenticated && user._id === event.createdBy) &&
```

Comment:
And while we are in this file, can you re-implement the commented code? I think it was almost functional but needs to be styled better.